### ISSUE TITLE
wanrnings in command mode and errors in hug-api mode 

### ISSUE THREAD

**Issue by shilik** [sugiyamath/sentiment_ja#2]
Hi, @sugiyamath 

I have followed your instructions for installation. After running the codes it yields some warning messages regarding tensorflow.
```
Using TensorFlow backend.
WARNING:tensorflow:From /home/shilik/venvs/hugserv/lib/python3.6/site-packages/tensorflow/python/keras/initializers.py:119: calling RandomUniform.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
WARNING:tensorflow:From /home/shilik/venvs/hugserv/lib/python3.6/site-packages/tensorflow/python/ops/math_grad.py:1250: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
[{'sentence': 'アフガニスタン東部ナンガルハル州ジャララバードで４日、現地で医療支援などの活動を続ける日本人医師、中村哲さん（７３）らが乗った車が武装勢力に襲撃された。ロイター通信などによると、中村さんは右胸に銃弾を受け、撃たれた直後は意識があったが、運ばれた病院で死亡した。', 'emotions': {'happy': '0.0', 'sad': '1.0', 'disgust': '3.0', 'angry': '0.0', 'fear': '32.0', 'surprise': '5.0'}}]
```

Is there way to get rid of these warnings?

When I use sentiment_ja with hug-api to make a webapi for my project.
Several errors occurred that prevented the app from running.
At first, I have the error message like the following.

``` 
mod_wsgi (pid=5595): Exception occurred processing WSGI script '/home/shilik/hug_api/sentija_api.py'.
 Traceback (most recent call last):
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/hug/api.py", line 502, in api_auto_instantiate
     return module.__hug_wsgi__(*args, **kwargs)
   File "falcon/api.py", line 274, in falcon.api.API.__call__
   File "falcon/api.py", line 269, in falcon.api.API.__call__
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/hug/interface.py", line 946, in __call__
     raise exception
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/hug/interface.py", line 917, in __call__
     self.call_function(input_parameters), context, request, response, **kwargs
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/hug/interface.py", line 839, in call_function
     return self.interface(**parameters)
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/hug/interface.py", line 129, in __call__
     return __hug_internal_self._function(*args, **kwargs)
   File "/home/shilik/hug_api/sentija_api.py", line 20, in sentimentja_api
     analyzer = Analyzer()
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/sentimentja-0.0.1f-py3.6.egg/sentimentja/__init__.py", line 18, in __init__
     self.model, self.graph = sentiment.load(join(path, "model_2018-08-28-15:00.h5"))
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/sentimentja-0.0.1f-py3.6.egg/sentimentja/sentiment.py", line 
 AttributeError: module 'tensorflow' has no attribute 'get_default_graph'
```

Then I changed the codes from

```
from keras.preprocessing.sequence import pad_sequences
from keras.models import load_model
```
to
```from tensorflow.keras.preprocessing.sequence import pad_sequences
from tensorflow.keras.models import load_model
```
The error of" AttributeError: module 'tensorflow' has no attribute 'get_default_graph'" disappeared.
But another error pop-up!

```
analyzer = Analyzer()

   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/sentimentja-0.0.1f-py3.6.egg/sentimentja/__init__.py", line 18, in __init__
     self.model, self.graph = sentiment.load(join(path, "model_2018-08-28-15:00.h5"))
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/sentimentja-0.0.1f-py3.6.egg/sentimentja/sentiment.py", line 22, in load
     return preds
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/saving.py", line 492, in load_wrapper
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/saving.py", line 584, in load_model
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/saving.py", line 274, in _deserialize_model
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/saving.py", line 627, in model_from_config
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/layers/__init__.py", line 168, in deserialize
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/utils/generic_utils.py", line 147, in deserialize_keras_object
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/sequential.py", line 298, in from_config
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/sequential.py", line 88, in __init__
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/legacy/interfaces.py", line 91, in wrapper
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/network.py", line 97, in __init__
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/network.py", line 304, in _init_subclassed_network
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/network.py", line 111, in _base_init
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/network.py", line 323, in __setattr__
   File "/home/shilik/venvs/hugserv/lib/python3.6/site-packages/Keras-2.3.0-py3.6.egg/keras/engine/base_layer.py", line 1215, in __setattr__
AttributeError: '_thread._local' object has no attribute 'value'
```

I have tried to upgrade/downgrade keras and tensorflow. But backend message keeps showing that sentiment_ja is not compatible with these versions except for tensorflow==1.14.0 & keras==2.3.0

I do want use your library in my project, is there any way to get these problems solved?
Thank you very much,

**Comment by sugiyamath** [comment]
Currently, this project supports "keras==2.3.0" and "tensorflow==1.14.0" only.
If you installed other versions of them, you need to use virtualenv to use the right versions.

By the way, if you want to disable tensorflow warnings, see the SO answers:
https://stackoverflow.com/questions/35911252/disable-tensorflow-debugging-information

In the future, I might be going to update this project to use the latest tensorflow versions, however sentiment_ja is not hard stuff to make by yourself.

If you want to create your own model, please use these steps:

1. search tweets by emoticons that corresponding to emotions.
2. scrape them.
3. annotate tweets automatically, by its emoticons. (This method is called "distant supervision".)
4. remove all emoticons from annotated tweet texts.
5. create a model in which inputs are each tweet texts, and outputs are annotated labels. (you can use any ML framework)
6. test the model by separated test data.

sentiment_ja uses "character-based CNN", however, there are better models like "subword-based CNN".

If you want to know the details, sentiment140 is a good resource.
http://cs.stanford.edu/people/alecmgo/papers/TwitterDistantSupervision09.pdf

**Comment by shilik** [comment]
Thank you very much for the advice.
As a final try, I have just tried to get rid of the error messages this
morning.
After removing both keras and tensorflow and reinstalling them again to the
certain version you mentioned above,
Fortunately, the codes yield output as expected when Apache server
restarted.
Though apache in the back-end  still throws out some warning messages
regarding some deprecated functions,
I have managed to get them corrected.
Anyway, I am grateful for your quick response as well as the great library,
I believe your library will be a feature of my project.

best regards.

shilik
